test/shared/src/main/scala/zio/test/DefaultTestReporter.scala

```diff
@@ -26,136 +27,139 @@ import zio.{Cause, _}
 
 import java.util.regex.Pattern
 import scala.annotation.tailrec
-import scala.util.Try
 
+// TODO Needs to be re-written or simply dropped for new streaming behavior
 object DefaultTestReporter {
-  def render[E](
-    executedSpec: ExecutedSpec[E],
+  def apply[E](testRenderer: TestRenderer, testAnnotationRenderer: TestAnnotationRenderer)(implicit
+    trace: ZTraceElement
+  ): TestReporter[E] = { (duration: Duration, executedSpec: ExecutionEvent) =>
+    // val rendered = testRenderer.render(render(executedSpec, true), testAnnotationRenderer)
+    // val stats    = testRenderer.render(logStats(duration, executedSpec) :: Nil, testAnnotationRenderer)
+    val rendered = List.empty
```

Comment:
Dead code?

Reply:
Leaving this in for the moment, so that I can scrutinize it in isolation here.
https://github.com/zio/zio/issues/6484
Added ticket # to comments